### PR TITLE
Updated link to validation document

### PR DESCRIPTION
**Pull request by anwara000**
Updated link to validation document

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/customer-key-tenant-level.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ For Exchange Online, you can create a DEP that encrypts one or more user mailbox
 
 ## Set up Customer Key at the tenant level (public preview)
 
-These steps are similar but not identical to the steps for setting up Customer Key at the application level. You should only use this public preview with test data in test tenants. Do not use this release with production data or in your production environment. If you already have a production deployment of Customer Key, use these steps to set up Customer Key at the tenant level in a test environment.
+These steps are similar but not identical to the steps for setting up Customer Key at the application level. You should only use this public preview with test data in test tenants. Do not use this release with production data or in your production environment. If you already have a production deployment of Customer Key, use these steps to set up Customer Key at the tenant level in a test environment. Once you have assigned a tenant level DEP to your tenant, you can start validation process and reachout to m365ck@microsoft.com with any questions or concerns. We have documented validation steps [here](https://aka.ms/CustomerKey/PublicPreviewValidation).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-tenant-level.md]:
```suggestion
These steps are similar but not identical to the steps for setting up Customer Key at the application level. You should only use this public preview with test data in test tenants. Do not use this release with production data or in your production environment. If you already have a production deployment of Customer Key, use these steps to set up Customer Key at the tenant level in a test environment. Once you have assigned a tenant level DEP to your tenant, you can start the validation process and reach out to m365ck@microsoft.com with any questions or concerns. You can also find documented validation steps in the public preview of [Validation Instructions for Data-at-rest Encryption for Microsoft 365](https://aka.ms/CustomerKey/PublicPreviewValidation).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@anwara-ms

**Comment by anwara000** [comment]
looks good to me.

**Comment by anwara000** [comment]
is it ready to be published today?

**Comment by JohanFreelancer9** [comment]
@anwara-ms - please commit my changes.

**Comment by anwara000** [comment]
I just did. All looking good to publish?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by anwara000** [comment]
Hi, this is still not published. Reminder- @KCCross

**Pull request merged**